### PR TITLE
Using go.rice for store Grafana Dashboard json

### PR DESCRIPTION
**Pull request by dmvolod**

### CODE REVIEW

File: pkg/controller/infinispan/resources/operatorconfig/rice-box.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
...eAll\": true,\n        \"label\": null,\n        \"multi\": true,\n        \"name\": \"caches\",\n        \"options\": [],\n        \"query\": \"query_result({job=~\\\"$cluster\\\", namespace=~\\\"$namespace\\\", __name__=~\\\"vendor_cache_manager_default_cache_.*_cluster_cache_stats_activations\\\"})\",\n        \"refresh\": 1,\n        \"regex\": \"/vendor_cache_manager_default_cache_(.*)_cluster_cache_stats_activations/\",\n        \"skipUrlSync\": false,\n        \"sort\": 0,\n        \"tagValuesQuery\": \"\",\n        \"tags\": [],\n        \"tagsQuery\": \"\",\n        \"type\": \"query\",\n        \"useTags\": false\n      }\n    ]\n  },\n  \"time\": {\n    \"from\": \"now-30m\",\n    \"to\": \"now\"\n  },\n  \"timepicker\": {\n    \"refresh_intervals\": [\n      \"10s\",\n      \"30s\",\n      \"1m\",\n      \"5m\",\n      \"15m\",\n      \"30m\",\n      \"1h\",\n      \"2h\",\n      \"1d\"\n    ],\n    \"time_options\": [\n      \"5m\",\n      \"15m\",\n      \"1h\",\n      \"6h\",\n      \"12h\",\n      \"24h\",\n      \"2d\",\n      \"7d\",\n      \"30d\"\n    ]\n  },\n  \"timezone\": \"\",\n  \"title\": \"Infinispan\",\n  \"uid\": \"U826WzBGz\",\n  \"version\": 35\n}"),
```
Comment (Crumby) [inline pkg/controller/infinispan/resources/operatorconfig/rice-box.go:17]: Having whole dashboard json file in single line doesn't look practical. Also doesn't it just duplicate `grafana_dashboard.json` file?

Comment (dmvolod) [reply pkg/controller/infinispan/resources/operatorconfig/rice-box.go:17]: This is generator called rice. File should be identical to original JSON. I will add generator calling script later and validate against env. Storing JSON as const as right now looks like a long string, not JSON formatted

Comment (dmvolod) [reply pkg/controller/infinispan/resources/operatorconfig/rice-box.go:17]: @Crumby I've validated on OCP env and Grafana Dashboard JSON seems to be looks like better, than JSON as string const
image

**Comment by ryanemerson** [comment]
@dmvolod +1 I like this approach a lot more than using a constant. It will scale much better as we add more resource files. 

Can you rebase, I just merged https://github.com/infinispan/infinispan-operator/pull/988 which updated the grafana json.

**Comment by wburns** [comment]
Sorry, I am not familiar, what benefit does rice.go provide when we still have the original dashboard json file?

**Comment by dmvolod** [comment]
> Can you rebase, I just merged #988 which updated the grafana json.

Yes, sure, thanks @ryanemerson

**Comment by ryanemerson** [comment]
> Sorry, I am not familiar, what benefit does rice.go provide when we still have the original dashboard json file?

@wburns It means that we can have the dashboard stored as a `.json` file in the local repo, instead of being defined in `.go` file as a constant. So it's easier to validate the json directly.

**Comment by wburns** [comment]
> > Sorry, I am not familiar, what benefit does rice.go provide when we still have the original dashboard json file?
> 
> @wburns It means that we can have the dashboard stored as a `.json` file in the local repo, instead of being defined in `.go` file as a constant. So it's easier to validate the json directly.

Ah, gotcha. Cool!

**Review by rigazilla — APPROVED** [review]
nice solution @dmvolod !

**Comment by dmvolod** [comment]
@rigazilla , @ryanemerson it would be nice to merge this PR before downstream release as this code will cause GarafanaDashboard CR update in the future

**Comment by ryanemerson** [comment]
@dmvolod Sorry about that. If you can rebase quick, I'll merge before we tag for downstream.

**Comment by dmvolod** [comment]
> If you can rebase quick, I'll merge before we tag for downstream.

Yes, sure, thanks @ryanemerson , done.

**Comment by ryanemerson** [comment]
Thanks @dmvolod

**Pull request merged**